packages/scandipwa/i18n/fa_IR.json

```diff
@@ -1,307 +1,121 @@
 {
-    "Something went wrong!": "با عرض پوزش خطایی رخ داده است!",
```

Comment:
Why did you remove all the translated strings and regenerated translation file? (I suppose it was by accident).

Reply:
Aleksandra asked me to, it was impossible to test for some reason